### PR TITLE
Update Getting Started Section to Use dapp-card-store

### PR DESCRIPTION
**Pull request by ivanlei**
# Summary
Update Getting Started Section to Use dapp-card-store

# Description
The card store dapp is considered a "more interesting" dapp because it uses Zoe. Let's update the instructions in getting started to use card store.

# Testing
Tested locally on clean checkout

### CODE REVIEW

File: main/guides/getting-started/README.md [hunk 1]
```diff
@@ -27,8 +27,8 @@ If `agoric` is not found, then you may need to add the parent directory of where
 This presentation is a good overview of the Agoric SDK setup process,
 though a few details are out of date:
 
-- node version: 12.x is too old; use 14.15.0 or higher
-- skip `git checkout hackathon-2020-11`; use the default `master` branch
+- node version: 12.x is too old; use 16.19.1 or higher
+- skip `git checkout hackathon-2020-11`; use the `community-dev` branch
```
Comment (Tyrosine22) [inline main/guides/getting-started/README.md:32]: Have you verified that Agoric now requires this later version of Node?

File: main/guides/getting-started/start-a-project.md [hunk 2]
```diff
@@ -96,32 +96,36 @@ cd ui && yarn start
...
-Once here, you will be asked to enable the dapp in your Agoric wallet.
+## Connect the Dapp to the Agoric Wallet
 
-![dapp fungible faucet ui - needs approval](./assets/agoric-ui-dapp-needs-approval.png)
+1. Once here, you will be asked to enable the dapp in your Agoric wallet.
 
-## Connect the Dapp to the Agoric Wallet
+![dapp card store ui - needs approval](./assets/must-enable-dapp.png)
+
+1. Navigate back to [http://127.0.0.1:8000](http://127.0.0.1:8000) and accept the Dapp's request to connect to your wallet.
```
Comment (Tyrosine22) [inline main/guides/getting-started/start-a-project.md:105]: We should be consistent in how we capitalize "dapp". I'm in favor of lowercase, but an argument could be made to capitalize, I suppose. (If it's good enough for the Germans, who are we to argue?)

File: main/guides/getting-started/start-a-project.md [hunk 3]
```diff
@@ -96,32 +96,36 @@ cd ui && yarn start
...
+   ![Proposed Offer](./assets/proposed-offer.png)
 
-Navigate back to [http://127.0.0.1:8000](http://127.0.0.1:8000) to approve the offer and collect the tokens.
+1. In the wallet, the offer will be in a `Pending` state while the auction for the card to complete. The auction takes up to 300 seconds.
 
-![agoric wallet - accept offer](./assets/agoric-wallet-approve-offer.png)
+   ![Pending Offer](./assets/pending-offer.png)
 
-Your wallet's FungibleFaucet purse will now hold 1000 FungibleFaucet tokens.
+1. In the wallet, your offer will transition to an `Accepted` state when the auction ends. Your `cardStore.Card` purse will now contain a card
```
Comment (Tyrosine22) [inline main/guides/getting-started/start-a-project.md:128]: This sentence needs a period.

**Review by Tyrosine22 — CHANGES_REQUESTED** [review]

File: main/guides/getting-started/README.md (hunk 1 above)
Comment (ivanlei) [reply main/guides/getting-started/README.md:32]: Yes. This is the requirement in agoric-sdk. See https://github.com/Agoric/agoric-sdk/blob/master/package.json#L36

**Review by Tyrosine22 — APPROVED** [review]

**Pull request merged**